test/test_invalid_headers.py

```diff
@@ -102,3 +102,8 @@ def test_range_of_acceptable_outputs(self, headers):
             assert headers == h2.utilities.validate_headers(headers)
         except h2.exceptions.ProtocolError:
             assert True
+
+    def test_invalid_pseudo_headers(self):
+        headers = ((b':custom', b'value'))
```

Comment:
I don't think this actually creates a tuple within a tuple. Consider changing this to a list with one tuple entry. That's why your test is failing.